packages/db/src/lib/granule.ts

```diff
@@ -45,3 +48,39 @@ export const upsertGranuleWithExecutionJoinRecord = async (
   }
   return [granuleCumulusId];
 };
+
+/**
+ * Get cumulus IDs for all executions associated to a set of granules
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction -
+ *  DB client or transaction
+ * @param {Array<Object>} granules - array of granules with collectionId and granuleId
+ * @param {Object} [collectionPgModel] - Collection PG model class instance
+ * @param {Object} [granulePgModel] - Granule PG model class instance
+ * @param {Object} [granulesExecutionsPgModel]
+ *   Granules/executions PG model class instance
+ * @returns {Promise<number[]>}
+ */
+export const getGranuleExecutionCumulusIds = async (
```

Comment:
```suggestion
export const getApiGranuleExecutionCumulusIds = async (
```

This is nitpicky, but if the function is going to except input in the form of a `collectionId`/`granuleId`, which are from the Dynamo/existing API structure, I think it's helpful